release-1.8/release_notes_draft.md

```diff
@@ -221,3 +221,24 @@ kind.
 * [alpha] Support ipvs mode for kube-proxy([#46580](https://github.com/kubernetes/kubernetes/pull/46580), [@haibinxie](https://github.com/haibinxie))
 
 ## External Dependencies
+Continuous integration builds have used the following versions of external
+dependencies, however, this is not a strong recommendation and users should
+consult an appropriate installation or upgrade guide before deciding what
+versions of etcd, docker or rkt to use.
+
+- Docker versions 1.11.2, 1.12.6, 1.13.1 (17.03.2) have been validated
```

Comment:
I am not sure what's in the bracket : 17.03.2

Can we remove this item from the list and add to the paragraph ?

Reply:
17.03.2 is a patch version of 1.13, but I changed this to list it separately.